source/systems/cloud_user_guide.rst

```diff
@@ -97,7 +97,7 @@ following diagram.
   .. tab-item:: Perform Computations
      :sync: compute
 
-      `Computation <https://parallelworks.com/docs/navigating-the-platform#compute>`__
+      `Computation <https://parallelworks.com/docs/navigating-the-platform>`__
```

Comment:
This link no longer exists.  PW just updated their documentation for a new "feature".  Please replace with https://parallelworks.com/docs/compute/logging-in-controller

Reply:
Not the first time I got caught this way. Thx